cider/tests/CiderNongroupbyAggTest.cpp

```diff
@@ -188,6 +188,7 @@ void testScalarTypeSingleKey(MockTable* table,
 }
 
 TEST_F(CiderNongroupbyAggArrowFormatTest, SingleKeyScalarTypeTest) {
+  GTEST_SKIP_("FIXME(haiwei): [POAE7-2511] Update verify test case of Non Groupby Agg");
```

Comment:
Shouldn't these TCs be resolved in this PR?

Reply:
This case has a large amount of changes, so i split a subtask to update it.